### DISCUSSION TITLE
How can I customize the number of digits for segment number?

### DISCUSSION THREAD

**Discussion by Ryjjjj45** [mifi/lossless-cut#1446] [Q&A]
If I set the export name as ${FILENAME}-${SEG_NUM} ${SEG_LABEL}${EXT}, how to set the ${SEG_NUM} as 2 digits? 
For example, if the amount of segments is less than 10, show the ${SEG_NUM} as 01-09, if the amount of segments is more than 10, show the ${SEG_NUM} as 10-99.
I try to add "0" before the ${SEG_NUM} now to make it show 01-09, but if the amount of segment is more than 10, I always need to rename the files manually. It's too much trouble, so I want to ask is there any method to solve it.

**Comment by mifi** [comment]
I have now implemented auto zero padding for SEG_NUM

**Reply by f34r1335** [reply]
Should this be working automatically? It doesn't work for me and I can't find an option in the settings for it either. Let me know, thanks.

**Reply by mifi** [reply]
You cannot customize but yes it should choose the right padding based on the number of segments total

**Reply by f34r1335** [reply]
How to troubleshoot this if it's not working? I tried exporting 11 segments, but 1-9 still are not padded. Will the padding also show in the filename preview if it works?

**Reply by mifi** [reply]
Yes it should. So you see 11 files in your output directory, each with the number 1 to 11, but none of them are padded? Could be a bug then

**Reply by mifi** [reply]
Yes it's a bug. thanks for reporting. will fix it now
